app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/IngredientsProductFragment.java

```diff
@@ -231,6 +300,10 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
         }
     }
 
+    private String trimLanguagePartFromString(String string) {
```

Comment:
Can you document this? I know it's a small method but I would like you to document what you use it for

Reply:
On it :+1:

Reply:
Update my PR here are the resultant Screenshots:
<br>
<img src="https://user-images.githubusercontent.com/12881364/37600871-9175ebf8-2bae-11e8-9b14-d81b78320a2c.png" height="600"/>
<img src="https://user-images.githubusercontent.com/12881364/37600872-91be8084-2bae-11e8-9c45-d20948976578.png" height="600"/>